taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -123,6 +138,37 @@ export const inject = (deps: InjectedDependencies) => {
 			);
 	};
 
+	const getFolderForScaffolding = async (
```

Comment:
I updated this to show message to the user properly. I believe that message is enough for the user to take action. If we throw, the calling code will log a more verbose message which can harm UX. The only problem is that from the caller's perspective, there is no way to know that some failure has occurred. For now, I don't see how that can be bad, but it can be useful to make sure all functions have inform us whether or not they have succeeded.